### ISSUE TITLE
Can the model trained by the original author be reused here?

### ISSUE THREAD

**Issue by 675492062** [szahid86/GenerativeFaceCompletion_Windows#1]
Can the model trained by the original website（https://github.com/Yijunmaverick/GenerativeFaceCompletion） be reused here?

**Comment by szahid86** [comment]
Yes. It reads the same prototxt (network definition) and caffe model (weights) file.

**Comment by 675492062** [comment]
Many thank you for your reply！But there is a problem that matlab is corrupted when i run the demo_face128.m.I really don't kown what has happend?

**Comment by 675492062** [comment]
my cl.exe is from vs2015

**Comment by 675492062** [comment]
Is it necessary to merge the recompiled Caffe version with this project? can we copy the matlab file from this project into recompied Caffe?

**Comment by szahid86** [comment]
@675492062 you first need to download all the dependencies mentioned in the readme file, and then compile the original Windows-Caffe on your system.
It is better that you run the mnist example after the compile and verify your installations.

Next, you need to copy the files from this repository and replace the original Windows-caffe files, and then compile once again.

I think you did not follow the step-by-step instructions.

**Comment by 675492062** [comment]
@szahid86 
I have done:
1)compiling the original Windows-Caffe on my system.                        OK
2)runing the cat classification example to verify this installation.          OK 
3)copying the files from this repository and replace the original Windows-caffe files ,and then comile once again.                                                                                             ERROR
I wonder if it's the caffe version.
